[2022-08-29 15:04:31,011] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-07-10T06:00:00+00:00 [queued]>
[2022-08-29 15:04:31,028] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-07-10T06:00:00+00:00 [queued]>
[2022-08-29 15:04:31,031] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:04:31,032] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-08-29 15:04:31,034] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:04:31,055] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): entry_task> on 2021-07-10 06:00:00+00:00
[2022-08-29 15:04:31,063] {standard_task_runner.py:52} INFO - Started process 10016 to run task
[2022-08-29 15:04:31,075] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-07-10T06:00:00+00:00', '--job-id', '1479', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmprcztmnlw', '--error-file', '/tmp/tmpb3cs_qpf']
[2022-08-29 15:04:31,079] {standard_task_runner.py:77} INFO - Job 1479: Subtask entry_task
[2022-08-29 15:04:31,185] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-07-10T06:00:00+00:00 [running]> on host efb24f1b01c6
[2022-08-29 15:04:31,272] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-07-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-10T06:00:00+00:00
[2022-08-29 15:04:31,275] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-29 15:04:31,277] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 2021_Q%-q.xlsx']
[2022-08-29 15:04:31,298] {subprocess.py:85} INFO - Output:
[2022-08-29 15:04:31,304] {subprocess.py:89} INFO - 2021_Q%-q.xlsx
[2022-08-29 15:04:31,307] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-29 15:04:31,345] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210710T060000, start_date=20220829T150431, end_date=20220829T150431
[2022-08-29 15:04:31,406] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 15:04:31,455] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
